src/CommunityToolkit.Maui.Core/Layouts/DockLayoutManager.shared.cs

```diff
@@ -107,17 +105,9 @@ public override Size ArrangeChildren(Rect bounds)
 
 			switch (dockLayout.GetDock(child))
 			{
-				case DockEnum.Left:
-				{
-					childX = x;
-					childY = y;
-					childHeight = height;
-					x += (childWidth + spacing.Width);
-					width -= (childWidth + spacing.Width);
-					break;
-				}
 				case DockEnum.Top:
 				case DockEnum.None:
+				default:
```

Comment:
Not sure if this should be at the end of the switch statement. But if the tests worked I'm ok with this here

Reply:
According to https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/statements/selection-statements

> The default case can appear in any place within a switch statement. Regardless of its position, the default case is always evaluated last and only if all other case patterns aren't matched.

I also tested this, and it seems to work Ok (I wasn't sure myself ;))